src/main/resources/rules/civil/calculations/thresholds/calculateThresholds.drl

```diff
@@ -41,6 +41,7 @@ rule "initialise income and asset thresholds"
 	thresholds.setCapitalUpperLimit(new BigDecimal("7999.99"));
 	thresholds.setGrossIncomeUpperLimit(new BigDecimal("2657.00"));
 	thresholds.setDisposableIncomeUpperLimit(new BigDecimal("733.00"));
+	thresholds.setDisposableIncomeUpperLimitDV(new BigDecimal("100888.00"));
```

Comment:
I think we should set this when we are setting the waiver. i.e. override the value it the waiver applies